Sources/MongoSwift/Operations/UpdateOperation.swift

```diff
@@ -0,0 +1,134 @@
+import mongoc
+
+internal protocol UpdateOperationOptions: Encodable {
+    var writeConcern: WriteConcern? { get }
+}
+
+/// Options to use when executing an `update` command on a `MongoCollection`.
+public struct UpdateOptions: UpdateOperationOptions {
```

Comment:
This feels like we reduce readability by having to look elsewhere that this struct still conforms to `Encodable`